tests/migration/ember-addon/steps/use-relative-paths/base-case.test.js

```diff
@@ -0,0 +1,163 @@
+import { useRelativePaths } from '../../../../../src/migration/ember-addon/steps/use-relative-paths.js';
+import { assertFixture, loadFixture, test } from '../../../../test-helpers.js';
+
+const fileMapping = new Map([
+  [
+    'addon/components/container-query.ts',
+    {
+      input: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        ``,
+        `import type {`,
+        `  Dimensions,`,
+        `  Features,`,
+        `  IndexSignatureParameter,`,
+        `  QueryResults,`,
+        `} from 'ember-container-query/modifiers/container-query';`,
+      ].join('\n'),
+
+      output: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        ``,
+        `import type {`,
+        `  Dimensions,`,
+        `  Features,`,
+        `  IndexSignatureParameter,`,
+        `  QueryResults,`,
+        `} from '../modifiers/container-query';`,
```

Comment:
Here, we're asserting that `useRelativePaths` ...

- Switches Ember's "magic" import path, `ember-container-query/modifiers/container-query`, to JavaScript's relative path, `../modifiers/container-query`
- Does not alter the rest of the file